distribution/packages/src/rpm/wazuh-indexer.rpm.spec

```diff
@@ -162,6 +162,7 @@ set -e
 if command -v systemctl >/dev/null && systemctl is-active %{name}.service >/dev/null; then
     echo "Stop existing %{name}.service"
     systemctl --no-reload stop %{name}.service
+    touch %{tmp_dir}/wazuh-indexer.restart
```

Comment:
Consider using [systemd-tmpfiles](https://www.baeldung.com/linux/systemd-tmpfiles-configure-temporary-files)

Reply:
Analysis in the comment https://github.com/wazuh/wazuh-indexer/pull/492#issuecomment-2441861588